xcodeproj/internal/xcodeproj_rule.bzl

```
@@ -461,12 +461,8 @@ targets.
         if xcode_target.product.is_resource_bundle and exclude_resource_bundles:
             # Don't create targets for resource bundles in BwB mode, but still
             # include their files if they aren't unfocused
-            focused_targets_extra_files.append(
-                (xcode_target.label, xcode_target.inputs.resources),
-            )
-            focused_targets_extra_folders.append(
-                (xcode_target.label, xcode_target.inputs.folder_resources),
-            )
+            focused_targets_extra_files.extend(xcode_target.inputs.resources.to_list())
+            focused_targets_extra_folders.extend(xcode_target.inputs.folder_resources.to_list())
```

```suggestion
            focused_targets_extra_files.extend(
                xcode_target.inputs.resources.to_list(),
            )
            focused_targets_extra_folders.extend(
                xcode_target.inputs.folder_resources.to_list(),
            )
```

I don't like the `to_list()` here. I think we can avoid it?

---

you mean `focused_targets_extra_folders.extend(xcode_target.inputs.folder_resources)`?